dice-where-downloader-lib/src/main/java/technology/dice/dicewhere/downloader/destination/local/LocalFileAcceptor.java

```diff
@@ -30,7 +31,7 @@ public LocalFileAcceptor(Path destination) {
 
   @Override
   public StreamConsumer<Void> getStreamConsumer(
-      MD5Checksum originalFileMd5, Instant originalFileTimestamp) {
+      MD5Checksum originalFileMd5, Instant originalFileTimestamp, boolean noMd5Check) {
```

Comment:
Can we invert this and not end up with the double negative? `md5Validated` or something?

Reply:
I would have to do it throughout the codebase (from the command line argument downwards), as this is the semantic everywhere.
But I'm with  you. Probably on a separate PR
The lack of unit tests also deters me from doing such a wide change on a whim.